### PR TITLE
Create Blip “dash-plotly”

### PR DESCRIPTION
**Pull request by arnaud-fossop**
Automatically generated by Decap CMS

### CODE REVIEW

**Review by julienblin — APPROVED** [review]
LGM!

File: src/content/blips/dash-plotly.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  * supports complex data visualizations (timeseries, 3D, etc.)
+
+  * interactive graphs and tables using React components
+
+  * can be embedded within your own API and return html page with react components
+
+  * Free version available
+
+
+  One drawback of using Dask is that some complex cases could require more support that is not provided with the free version. The enterprise offering includes enterprise features (security, RBAC, visualization workload management, etc.) and support but the pricing is not published on the website.
```
Comment (rimthong) [inline src/content/blips/dash-plotly.md:21]:
```suggestion
  One drawback of using Dash is that some complex cases could require more support that is not provided with the free version. The enterprise offering includes enterprise features (security, RBAC, visualization workload management, etc.) and support but the pricing is not published on the website.
```

**Review by rimthong — CHANGES_REQUESTED** [review]
Small typo. Also feels a bit lightweight. I guess it's ok for things in `assess` but it feels like we should provide more value / analysis.

**Comment by arnaud-fossop** [comment]
> Small typo. Also feels a bit lightweight. I guess it's ok for things in `assess` but it feels like we should provide more value / analysis.

I added information on why we use it for SynapseBio: we had to visualize time series that were stored in special data format, so requiring a specific pre-processing. I hope that shows the value you were talking about.

**Review by jorge-vasquez-acn — APPROVED** [review]

**Pull request merged**